File: Python/Estrutura sequencial/consumo.py
# Variáveis
# Em Python não é necessário fazer esse passo...
distanciaPercorrida: int
combustivelGasto: float; consumoMedio: float

# Entrada de dados
distanciaPercorrida = int(input("Distância percorrida: "))
combustivelGasto = float(input("Combustível gasto: "))

# Artíficio matemático
consumoMedio = distanciaPercorrida / combustivelGasto

# Saída de dados
print(f"O consumo médio é de {consumoMedio:.3f}")

File: C#/Estrutura sequencial/consumo.cs
// Variáveis
int distanciaPercorrida;
double combustivelGasto, consumoMedio;

// Entrada de dados
Console.Write("Distância percorrida: ");
distanciaPercorrida = int.Parse(Console.ReadLine());
Console.Write("Combustível gasto: ");
combustivelGasto = double.Parse(Console.ReadLine());

// Algoritmo matemático
consumoMedio = distanciaPercorrida / combustivelGasto;

// Saída de dados
Console.Write("O consumo médio foi de " + consumoMedio.ToString("F3") + "km.");